src/Management/Model/Item.php

```diff
@@ -9,7 +9,7 @@
  */
 class Item implements ModelInterface
 {
-    const SETTED_FIELDS = ['id', 'df_grouping_id', 'df_group_leader', 'df_manual_boost', 'categories', 'best_price'];
+    const SETTED_FIELDS = ['title', 'description', 'link', 'image_link', 'availability', 'price', 'sale_price', 'brand', 'gtin', 'mpn'];
```

Comment:
id field not required?

Reply:
yes, the id is carried by the model itself, this list is a list of possible additional fields that an item may have.
If you expand the model information, you can see the fixed fields shown, among them the id.